### DISCUSSION TITLE
fixing-nginx-upstream-sent-too-big-header-error-when-running-an-ingress-controller-in-kubernetes/

### DISCUSSION THREAD

**Discussion by andrewlock** [andrewlock/blog-comments#251] [Announcements]
# fixing-nginx-upstream-sent-too-big-header-error-when-running-an-ingress-controller-in-kubernetes/

This post describes how to fix the error "upstream sent too big header while reading response header from upstream" when using an Nginx ingress in Kubernetes

https://andrewlock.net/fixing-nginx-upstream-sent-too-big-header-error-when-running-an-ingress-controller-in-kubernetes/

[a comment by an automated account is not shown]

**Reply by moutansos** [reply]
I found this as well! Way simpler and easier to just add the annotations haha

[a comment by an automated account is not shown]
